wagn/rails/controllers/card_controller.rb

```diff
@@ -248,7 +243,9 @@ def show view = nil, status = 200
     else
       args = { :text=>result, :status=>status }
       args[:content_type] = 'text/text' if format == :file
-      render args
+      card.run_callbacks :render_view do
```

Comment:
re read: you're completely right.  read is an "action", so it can't be an event.

re view -- but there might be many views, even for the one card.   

Seems like we might want to give "page" a formal meaning at some point.

Oh well, guess I can't do better yet.